backend/config/geneinfo-api.yml

```diff
@@ -0,0 +1,72 @@
+openapi: 3.0.3
+info:
+  version: "1.0.0"
+  title: Chan Zuckerburg Initiative cellxgene Gene Info API
+  description: >-
+    This API is for internal use to access information from NCBI's E-Utilities API about a given gene.
+servers:
+  - description: Local
+    url: /
+  - description: Production environment
+    url: https://api.cellxgene.cziscience.com/
+  - description: Development environment
+    url: https://api.dev.single-cell.czi.technology/
+  - description: Staging environment
+    url: https://api.staging.single-cell.czi.technology/
+
+paths:
+  /v1/geneinfo:
+    get:
+      summary: Given a gene ensembl ID, returns that gene's summary information, full name, synonyms, and NCBI URL. 
+      description: >-
+      tags:
+        - geneinfo
+      operationId: backend.geneinfo.api.v1.geneinfo
```

Comment:
nit but I think it might be worth writing it as `gene_info` instead of geneinfo to follow the general pattern in the code